src/components/common/ModeratorList.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Text } from "native-base";
+import AvatarUsername from "./AvatarUsername";
+
+interface IProps {
+  item: any;
+}
+
+function moderatorList({ item }: IProps) {
```

Comment:
We should use pascal case for component names. Also, I think it might be better to call this "ModeratorListItem" since it isn't actually the whole list.

Reply:
updated